qemu/tests/cfg/x86_cpu_model.cfg

```diff
@@ -37,6 +37,12 @@
             only HostCpuVendor.amd
             flags = "vme sse2 sse fxsr mmx clflush pse36 pat cmov mca pge mtrr sep apic cx8 mce pae msr tsc pse de  popcnt cx16 lm nx syscall misalignsse sse4a abm lahf_lm fpu"
             model_pattern = "AMD Opteron 23xx \(Gen 3 Class Opteron%s\)"
+        - Cooperlake:
+            only HostCpuVendor.intel
+            flags = "avx512_bf16 stibp arch_capabilities"
+            model_pattern = "Intel Xeon Processor \(Cooperlake%s\)"
+            check_cmd = "grep . /sys/devices/system/cpu/vulnerabilities/%s"
```

Comment:
`cat /sys/devices/system/cpu/vulnerabilities/%s` might be more efficient.

Reply:
> `cat /sys/devices/system/cpu/vulnerabilities/%s` might be more efficient.

Thanks and update.